src/components/Button/Button.scss

```diff
@@ -1,78 +1,156 @@
 @import '../../colours.scss';
 
+@mixin button-styling($bg, $active-bg, $dis-bg, $dis-c, $focus-bg, $hover-bg, $c: default) {
+  background-color: $bg;
+
+  @if $c == 'default' {
+    color: $rpf-text-black;
+
+    svg {
+      fill: $rpf-text-black;
+    }
+  } @else {
+    color: $c;
+
+    svg {
+      fill: $c;
+    }
+  }
+
+  &:active {
+    background-color: $active-bg;
+    @content;
+  }
+
+  &:disabled {
+    background-color: $dis-bg;
+    color: $dis-c;
+  
+    svg {
+      fill: $dis-c;
+    }
+  }
+
+  &:focus-visible {
+    background-color: $focus-bg;
+    @content;
+  }
+  
+  &:hover {
+    background-color: $hover-bg;
+    @content;
+  }
+}
+
 .btn {
   align-items: center;
-  border-radius: 5px;
-  border-width: 0;
-  cursor: pointer;
+  border-radius: 8px;
+  border: 3px solid transparent;
   display: inline-flex;
   font-family: sans-serif;
   font-size: inherit;
-  font-weight: var(--font-weight-regular);
-  justify-content: center;
+  font-weight: var(--font-weight-bold);
   margin: 6px;
   padding: 10px;
   position: relative;
   text-align: center;
   text-decoration: none;
   width: fit-content;
 
+  &:focus-visible {
+    border: 3px solid $rpf-brand-raspberry;
+    outline: none;
+  }
+
   svg {
     margin-right: var(--spacing-1);
   }
 
+  &:disabled {
+    background-color: $editor-grey;
```

Comment:
Can we make this a colour from the new palette if possible? (I think the only example of this being used at the moment is the stopping button... (Put
```python
from time import sleep
sleep(10)
```
in the editor and run, then click stop to see the stopping button

Reply:
Done in [8a13902](https://github.com/RaspberryPiFoundation/editor-ui/pull/296/commits/8a1390255e494c4e46d3b2dbead00a47b2887213)